src/pages/login/Login.jsx

```diff
@@ -0,0 +1,109 @@
+import MailOutlinedIcon from "@mui/icons-material/MailOutlined";
+import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
+import * as yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+import "./Login.css";
+import {
+  Box,
+  Button,
+  Grid,
+  InputAdornment,
+  TextField,
+  Typography,
+} from "@mui/material";
+import google from "/src/imgs/gmail.png";
+
+const Login = () => {
+  const schema = yup.object().shape({
+    email: yup.string().email("it must be a e-mail").required("insert value"),
+    password: yup
+      .string()
+      .min(4, "It must have 4 characters")
+      .max(20, "It must be less than 20 characters")
+      .required("insert value"),
+  });
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(schema),
+  });
+
+  const onSubmit = (data) => {
+    window.location.href = "/";
```

Comment:
please change this implementations, as I told you before you need to redirect user to /home, not to /. You need to use react router dom to navigate.